### PR TITLE
Wh01 m125 toa01a01 tobbbb

### PR DESCRIPTION
**Pull request by aminghiasi**
This pull request will add "Wh01 m125 toa01a01 tobbbb" Madgraph cards for central production. We have already produced private MC samples using these cards and gave two updates in the HiggsExo meeting:
https://indico.cern.ch/event/668522/contributions/2741419/attachments/1537379/2409199/haa4b_georgia091017.pdf
https://indico.cern.ch/event/663418/contributions/2709637/attachments/1521025/2396927/haa4b_higgsexo110917.pdf

### CODE REVIEW

**Comment by perrozzi** [comment]
please check these instructions to cleanup the datacards
https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

**Comment by aminghiasi** [comment]
Hi Pietro,

I removed the 2016 files to create another PR for them. Also, I applied the checks you sent to me.

Cheers,
Amin

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.1.2                 2014-07-03         *
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_proc_card.dat]: please remove the banners from the proc_card

**Comment by aminghiasi** [comment]
Hi Perrozzi,
I removed the banners from the production cards.
Cheers,
Amin

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+ 15   = xqcut   ! minimum kt jet measure between partons
+#*********************************************************************
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+#   F  = use_syst      ! Enable systematics studies
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_run_card.dat]: remove these lines

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+   F  = cut_decays      ! Enable systematics studies   
+   T = use_syst
+#
+#**************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_run_card.dat]: @kdlong @qliphy are these lines outdated or need to be kept?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_run_card.dat (hunk 2 above)
Comment (aminghiasi) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_run_card.dat]: I removed the lines you mentioned.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+ -1    =  ktdurham        
+ 0.4  =  dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+# Parameter of the systematics study
+#  will be used by SysCalc (if installed)
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_run_card.dat]: I believe syscalc is obsolete also in this case, right @kdlong @qliphy ?

Comment (aminghiasi) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh01_M125_Toa01a01_Tobbbb_M12to60_LO_MLM/Wh01_M125_Toa01a01_Tobbbb_M12_LO_MLM/Wh01_M125_Toa01a01_M12_Tobbbb_run_card.dat]: Done!

**Comment by perrozzi** [comment]
I am afraid the wrong lines were removed. you need to remove only
```
# Parameter of the systematics study
#  will be used by SysCalc (if installed)
#**************************************                                  
#
0.5 1 2 = sys_scalefact  # factorization/renormalization scale factor
0.5 1 2 = sys_alpsfact  # \alpha_s emission scale factors
30 50 = sys_matchscale # variation of merging scale
# PDF sets and number of members (0 or none for all members).
CT10nlo.LHgrid = sys_pdf # matching scales
# MSTW2008nlo68cl.LHgrid 1  = sys_pdf
```

**Comment by aminghiasi** [comment]
I think I removed the correct lines this time!

**Comment by perrozzi** [comment]
yes but you still didn't follow all the suggestions in 
https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T
(there are double imports, only the last one matters...)

**Comment by aminghiasi** [comment]
Thanks for looking carefully at my cards. I removed the first import from the production cards

**Comment by perrozzi** [comment]
things look ok to me now

**Pull request merged**